### PR TITLE
Update members.json

### PR DESCRIPTION
**Pull request by Matt-code-d**

### CODE REVIEW

**Comment by Matt-code-d** [comment]
Added my website

**Comment by whitep4nth3r** [comment]
Hi! Please could you put your entry at the bottom of the list to keep entries in chronological order?

Thanks!

**Comment by whitep4nth3r** [comment]
Love your website, by the way!

**Comment by Matt-code-d** [comment]
> Hi! Please could you put your entry at the bottom of the list to keep entries in chronological order?
> 
> Thanks!

Of coure, i'm sorry , i made a mistake in pasting

File: data/members.json [hunk 1]
```diff
@@ -161,5 +156,10 @@
     "name": "Tori Hall",
     "feed": null,
     "url": "https://toriii.neocities.org/"
-  }
+  },
+  {
+    "name": "Matteo Santoro Dev",
+    "feed": null,
+    "url": "https://matteosantoro.dev/"
+  },
```
Comment (whitep4nth3r) [inline data/members.json:164]: Please remove this comma, it will break everything!

**Comment by Matt-code-d** [comment]
> Love your website, by the way!

Thank U very much!

**Pull request merged**